### PR TITLE
Llazarek/feature/octopos_driver

### PR DESCRIPTION
**Pull request by LLazarek**

### CODE REVIEW

File: include/Optional.hpp [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+     * return the given default.
+     *
+     * @param default_value The default value to return in case of None.
+     * @return The value in this Option or `default_value`.
+     */
+    T getDefault(T default_value) {
+        return empty ? default_value : x;
+    }
+
+    // throws: std::runtime_error
```
Comment (DarrienG) [inline include/Optional.hpp]: Random comment?

File: src/makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+main: Main.cpp octopOS_driver.cpp ../include/*.h* \
```
Comment (DarrienG) [inline src/makefile]: 🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔

**Review by DarrienG — DISMISSED** [review]

**Comment by LLazarek** [comment]
@JoshuaHassler CI stalled forever?

**Review by JoshuaHassler — APPROVED** [review]

**Comment by LLazarek** [comment]
🎉

**Pull request merged**